packages/framework/ai-collab/src/test/explicit-strategy/agentEditingReducer.spec.ts

```diff
@@ -380,84 +381,207 @@ describe("applyAgentEdit", () => {
 		});
 	});
 
-	it("modify edits", () => {
-		const tree = factory.create(
+	describe("modify edits", () => {
+		let tree = factory.create(
```

Comment:
Nit: slightly better if we just declare these here but don't initialize them, since the `beforeEach` will run anyway and overwrite them before the first test runs.